database/migrations/20210317210236-geography.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db, callback) {
+  db.runSql(
+    'ALTER TABLE treetracker.tree ADD COLUMN estimated_geographic_location GEOGRAPHY(Point)',
+    callback,
+  );
+  db.runSql(
+    'ALTER TABLE treetracker.capture ADD COLUMN estimated_geographic_location GEOGRAPHY(Point)',
```

Comment:
I was thinking that you need to do GEOGRAPHY(POINT,4326), to specify the projection, but turns out 4326 is the default so you are good here.

Reply:
I will include the SRID just to be specific.